src/consensus/read_validator.rs

```diff
@@ -96,7 +98,41 @@ impl ReadValidator {
         }
     }
 
+    fn verify_signatures(value: &proto::DecidedValue) -> bool {
```

Comment:
Please add a test for this

Reply:
Added signatures to all tests here https://github.com/farcasterxyz/snapchain/pull/280/commits/b91169a89dae9b948190985ba8ccd8a44bb7b97b

Let me know if you want something to further test this function, could do a unit test to make sure unsigned commits are skipped